### PR TITLE
remove cypress tests that are covered by integration tests: seo & metadata

### LINKED ISSUE #6648: Migrate SEO / metadata tests from Cypress to integration
**Is your feature request related to a problem? Please describe.**
We would like to start migrating tests from Cypress to integration, in small manageable chunks.
We have identified a number of "SEO / metadata" Cypress tests which already exist in integration tests, which can therefore be removed. 

**Describe the solution you'd like**
The following tests run for every configured URL in the Cypress config:
https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/testsForAllPages.js#L21
https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/testsForAllCanonicalPages.js#L19
https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/testsForAllAMPPages.js#L14

Many of these assertions are already run as part of integration tests:
https://github.com/bbc/simorgh/blob/latest/src/integration/common/SEO.js#L5
https://github.com/bbc/simorgh/blob/latest/src/integration/common/core.amp.js#L2
https://github.com/bbc/simorgh/blob/latest/src/integration/common/core.canonical.js#L2

- If the test already exists in integration tests, it should be removed from the cypress suite
- If the test does not already exist in integration tests, it should be added to integration tests and removed from the cypress suite

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #6648 

**Overall change:**
Removed a number of tests already covered by the integration tests. Added one test (apple touch icon)  to integration tests.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added labels to this PR for the relevant pod(s) affected by these changes
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/integration/pages/testsForAllPages.js [hunk 1]
```diff
@@ -18,278 +18,126 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
...
-
-                const cpsPageTypes = [
-                  mediaAssetPageType,
-                  photoGalleryPageType,
-                  storyPageType,
-                ];
-
-                const { indexImage } = body.promo;
-                const imagePath = indexImage ? indexImage.path : null;
+      if (pageType !== 'errorPage404' && pageType !== 'mostReadPage') {
```
Comment (karinathomasbbc) [inline cypress/integration/pages/testsForAllPages.js]: I believe that all of this can be removed?

Comment (oluoluoxenfree) [reply cypress/integration/pages/testsForAllPages.js]: As in the `if` statement? @karinathomasbbc

Comment (karinathomasbbc) [reply cypress/integration/pages/testsForAllPages.js]: possibly the whole thing... let's catch up tomorrow!

File: cypress/integration/pages/testsForAllPages.js [hunk 2]
```diff
@@ -17,281 +15,6 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
   variant,
 }) => {
   describe(`Running testsForAllPages for ${service} ${pageType}`, () => {
```
Comment (karinathomasbbc) [inline cypress/integration/pages/testsForAllPages.js:14]: Wohoo! All gone!

File: src/integration/common/SEO.js [hunk 3]
```diff
@@ -192,5 +192,25 @@ export default () => {
...
+        const url = script.getAttribute('href');
+        const sizes = script.getAttribute('sizes');
+
+        it('should be in the document', () => {
+          expect(script).toBeInTheDocument();
+        });
+
+        it(`should match attributes`, () => {
+          expect(`${url}`).toMatchSnapshot();
+          expect(`${sizes}`).toMatchSnapshot();
```
Comment (karinathomasbbc) [inline src/integration/common/SEO.js]: Can we maybe separate these into 2 separate assertions?

```suggestion
        it(`should match url`, () => {
            expect(`${url}`).toMatchSnapshot();
          };
          it(`should match size`, () => {
            expect(`${sizes}`).toMatchSnapshot(); 
          };
          expect(`${sizes}`).toMatchSnapshot();
```

File: src/integration/common/SEO.js [hunk 4]
```diff
@@ -192,5 +192,25 @@ export default () => {
         });
       });
     });
+
+    describe('Apple Touch Icon', () => {
```
Comment (karinathomasbbc) [inline src/integration/common/SEO.js:196]: Do we also need to update the snapshots to include these?

**Review by karinathomasbbc — APPROVED** [review]
Looks great, thank you for doing this! 👍

File: src/integration/common/SEO.js (hunk 3 above)
Comment (karinathomasbbc) [reply src/integration/common/SEO.js]: Ignore this - I have a slightly better suggestion :)

File: src/integration/common/SEO.js [hunk 5]
```diff
@@ -192,5 +192,25 @@ export default () => {
...
+
+      touchIconScript.forEach(script => {
+        const url = script.getAttribute('href');
+        const sizes = script.getAttribute('sizes');
+
+        it('should be in the document', () => {
+          expect(script).toBeInTheDocument();
+        });
+
+        it(`should match attributes`, () => {
```
Comment (karinathomasbbc) [inline src/integration/common/SEO.js:209]: Could we try something like this (~stolen~ borrowed from another PR!) - https://github.com/bbc/simorgh/blob/2bd4daaa2962fbf91a59c1f02a277272ff3bdfed/src/integration/common/footer.js#L42

```suggestion
        it(`should match attributes`, () => {
          expect({
          url,
          sizes
          }).toMatchSnapshot();
        };
```

**Review by ryanmccombe — APPROVED** [review]
👍

File: cypress/integration/pages/testsForAllPages.js [hunk 6]
```diff
@@ -16,281 +15,31 @@ export const testsThatFollowSmokeTestConfigforAllPages = ({
...
+          appConfig[config[service].name][variant].translations.consentBanner
+            .cookie.title,
+        );
+        cy.contains(
+          appConfig[config[service].name][variant].translations.consentBanner
+            .cookie.reject,
+        );
+        cy.contains(
+          appConfig[config[service].name][variant].translations.consentBanner
+            .cookie.accept,
```
Comment (karinathomasbbc) [inline cypress/integration/pages/testsForAllPages.js]: This was removed as part of https://github.com/bbc/simorgh/pull/6925 (which was merged earlier today hence merge conflict 😿 ), so also needs to be removed here.

**Pull request merged**